src/css/styles/_top.scss

```diff
@@ -48,12 +59,46 @@
   color: #fff;
 }
 
-.nav-menu__lang {
-  padding: 1rem;
+.c-lang {
+  position: absolute;
+  top: 1rem;
+  right: 0;
+
+}
+
+.c-lang__toggle {
+  padding: 1rem 0.5rem 0.5rem !important;
+}
```

Comment:
![image](https://user-images.githubusercontent.com/789841/45746828-6d090980-bc04-11e8-8a2f-ff6bc23e1021.png)

- [ ] `v` should expand language menu